### PR TITLE
Programming Workout talk by Michał Taszycki

### PR DESCRIPTION
**Pull request by mehowte**
Our tools are becoming ever more efficient...  
Command line tools are becoming obsolete...  
Programmers today don't need to touch-type...  
Using the mouse to copy and paste is perfectly fine...  
You can always look up those design patterns on the web...  
Your IDE can do many things for you, so why do you even need to think?

Can you feel it? Can you feel that this is TRUE?

Then **stop** being UNPROFESSIONAL and **think again**!

In this talk, I'm going to convince you that learning seemingly obsolete skills can have huge impact on your productivity.
I will show you how these skills and other seemingly unimportant factors can impact your career.
I will help you find ways to improve these skills in order to become a better programmer.
I will also show you tools that can facilitate this process.

In other words, **I will show you how programmers WORK OUT**. 

You will either leave this talk with a strong resolution to level up, or curl up in your comfort zone with your lovely mouse and IDE.

### CODE REVIEW

**Comment by psyho** [comment]
:+1:

**Comment by mateuszzawisza** [comment]
:+1: Already seen some parts of it and it is a great idea :)

**Comment by konieczkow** [comment]
+1

**Comment by kstarski** [comment]
+1, :punch: workout :muscle:

**Comment by nbartlomiej** [comment]
:+1:

**Comment by roytomeij** [comment]
Saw this talk before and I liked it! Should get started with working out soon too... :P

**Comment by szemek** [comment]
:+1:

**Comment by domininik** [comment]
go!

**Comment by szarski** [comment]
+1

**Comment by sebastiansito** [comment]
:+1:

**Comment by yundt** [comment]
+1

**Comment by agatkamazur** [comment]
Programming workout FTW!

**Comment by marcinb** [comment]
+1

**Comment by mariuszlusiak** [comment]
:+1:

**Comment by nazgob** [comment]
+1

**Comment by skade** [comment]
+1

**Comment by peterberkenbosch** [comment]
+1

**Comment by tlossen** [comment]
+1 

(missed in at wroc_love.rb, would like a second chance .....)

**Comment by jstorimer** [comment]
+1

**Comment by Stanley** [comment]
:+1:

**Comment by rodzyn** [comment]
+1

**Comment by osegrums** [comment]
+1

**Comment by DouweM** [comment]
+1
